packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 45.3.0 - February 22, 2019
+
+### Added
+
+#### salesforcedx-vscode-core
+
+- Add flag to control initial sObject refresh ([PR #1079](https://github.com/forcedotcom/salesforcedx-vscode/pull/1079))
+
+### Fixed
+
+#### salesforcedx-vscode
+
+- Updates to search and analytics on the documentation site ([PR #1074](https://github.com/forcedotcom/salesforcedx-vscode/pull/1074))
```

Comment:
@ruthemmanuelle not sure if we should use documentation site or GitHub Pages, what should we call it ?